test/stdlib/LazyCollectionPlus.swift

```diff
@@ -1,5 +1,5 @@
 // RUN: %empty-directory(%t)
-// RUN: %target-build-swift %s -o %t/a.out3 -swift-version 3 && %target-run %t/a.out3
+// RUN: %target-build-swift %s -o %t/a.out -swift-version 3 && %target-run %t/a.out
```

Comment:
Should this be -swift-version 4?

Reply:
ah in fact the whole test should just go, we don't support this beyond Swift 3